libs/net/dal/Migrations/1.0.54/Up/PostUp/01-Schedule.sql

```diff
@@ -0,0 +1,132 @@
+DO $$
+BEGIN
+
+INSERT INTO public.schedule (
+    "name"
+    , "description"
+    , "is_enabled"
+    , "delay_ms"
+    , "run_on"
+    , "start_at"
+    , "stop_at"
+    , "repeat"
+    , "run_on_week_days"
+    , "run_on_months"
+    , "day_of_month"
+    , "created_by"
+    , "updated_by"
+    , "version"
+) VALUES (
+  'TNO 1.0 - AudioVideo Content - HISTORIC' -- name
+  , '' -- description
+  , true -- is_enabled
+  , 30000 -- delay_ms
+  , NULL -- run_on
+  , '00:00:00' -- start_at
+  , '23:59:59' -- stop_at
+  , false -- repeat
+  , 127 -- run_on_week_days
+  , 0 -- run_on_months
+  , 0 -- day_of_month
+  , '' -- created_by
+  , '' -- updated_on
+  , '0'
+) ON CONFLICT DO NOTHING;
```

Comment:
I think its the path of least resistance for doing an `insert`.  If you don't do this, it opens up a can of worms about what the alternative response should be.  If we have to check every `insert` to see if it should be an `update` it will be a real pain.